docs/reference/mapping/types/semantic-text.asciidoc

```diff
@@ -5,4 +5,134 @@
+When the first document is indexed, the `inference_id` will be used to generate underlying indexing structures for the field.
+
+
+[discrete]
+[[auto-text-chunking]]
+==== Automatic text chunking
+
+{infer-cap} services have a limit on the amount of text they can process.
+To allow for large amounts of text to be used in semantic search, `semantic_text` automatically generates smaller passages if needed called _chunks_.
+
+Each chunk will include the text subpassage and its corresponding embedding generated from it.
+When querying, the individual passages will be automatically searched for each document, and the score combined to obtain relevant results.
+
+
+[discrete]
+[[semantic-text-structure]]
+==== `semantic_text` structure
+
+Once ingested, a `semantic_text` field will have the following structure:
+
+[source,console-result]
+------------------------------------------------------------
+"inference_field": {
+  "text": "these are not the droids you're looking for", <1>
+  "inference": {
+    "inference_id": "my-elser-model", <2>
+    "model_settings": { <3>
+      "task_type": "sparse_embedding"
+    },
+    "chunks": [ <4>
+      {
+        "text": "these are not the droids you're looking for",
+        "embeddings": {
+		(...)
+        }
+      }
+    ]
+  }
+}
+------------------------------------------------------------
+// TEST[skip:TBD]
+<1> The field will become an object structure to accommodate both the original
+text and the inference results.
+<2> The `inference_id` used to generate the embeddings.
+<3> Model settings, including the task type and dimensions/similarity if
+applicable.
+<4> Inference results will be grouped in chunks, each with its corresponding
+text and embeddings.
+
+Refer to <<semantic-search-semantic-text,this tutorial>> to learn more about
+semantic search using `semantic_text` and `semantic_query`.
+
+
+[discrete]
+[[custom-indexing]]
+==== Customizing `semantic_text` indexing
+
+`semantic_text` uses defaults for indexing data based on the {infer} service
+specified. It enables you to quickstart your semantic search by providing
+automatic {infer} and a dedicated query so you don't need to provide further
+details.
+
+In case you want to customize data indexing, use the
+<<sparse-vector,`sparse_vector`>> or <<dense-vector,`dense_vector`>> field
+types and create an ingest pipeline with an
+<<inference-processor, {infer} processor>> to generate the embeddings.
+<<semantic-search-inference,This tutorial>> walks you through the process.
+
+
+[discrete]
+[[copy-to-support]]
+==== `copy_to` support
+
+The `semantic_text` field type can be the target of
+<<copy-to,`copy_to` fields>>. This means you can use a single `semantic_text`
+field to copy the values of other fields. Each value has its embeddings
+calculated separately; each field value is a separate chunk in the resulting
+embeddings.
+
+This imposes some restrictions on updating documents with `semantic_text`:
```

Comment:
We don't support update via script when an index contains a `semantic_text` field, should we add that here?

Reply:
I think copy_to deserves its own section, it's useful for users to understand the possibility to use this. But I'm open to suggestions and @szabosteve criteria here.

Reply:
I agree to have a dedicated section for `copy_to`. If we have more restrictions to list that are unrelated to `copy_to`, then we should add a section for restrictions. If we have only this one (update via script is not supported), then it's better to just add an IMPORTANT or NOTE callout with the info. WDYT?

Reply:
It makes sense to use a NOTE here from my PoV. Thanks @szabosteve !